scripts/clear_all.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Uncomment the desired network
+export NEAR_ENV=testnet
+# export NEAR_ENV=mainnet
+# export NEAR_ENV=guildnet
+# export NEAR_ENV=betanet
+
+export FACTORY=testnet
+# export FACTORY=near
+# export FACTORY=registrar
+
+if [ -z ${NEAR_ACCT+x} ]; then
+  export NEAR_ACCT=you.testnet
+else
+  export NEAR_ACCT=$NEAR_ACCT
+fi
+
+export CRON_ACCOUNT_ID=cron.$NEAR_ACCT
+export COUNTER_ACCOUNT_ID=counter.$NEAR_ACCT
+export AGENT_ACCOUNT_ID=agent.$NEAR_ACCT
+export USER_ACCOUNT_ID=user.$NEAR_ACCT
+export CRUD_ACCOUNT_ID=crud.$NEAR_ACCT
+export DAO_ACCOUNT_ID=dao.sputnikv2.testnet
+
+# clear and recreate all accounts
```

Comment:
doesn't recreate, but I see that's done in the create_and_deploy script 👍🏼

Reply:
Correct, just copy pasta :D